docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+```petab1
+@parameters σ
+obs_P = PEtabObservable(P, σ)
+```
+and then let the parameter *σ* vary between different [simulation conditions](@ref petab_simulation_conditions). Alternatively we could let the noise scale linearly with the species' concentration:
+```petab1
+obs_P = PEtabObservable(P, 0.05P)
+```
+
+PEtab.jl assumes that noise is normally distributed (with a standard deviation equal to the second argument provided to `PEtabObservable`). The only other (currently implemented) noise distribution is log-normally distributed noise, which is designated through the `transformation=:log` argument:
+```petab1
+obs_P = PEtabObservable(P, σ; transformation=:log)
+```
+
+## [Additional features: Parameters](@id petab_parameters)
+
+### [Known parameters](@id petab_parameters_known)
+In our previous example, all parameters were unknowns that we wished to fit to the data. If any parameters have known values, it is possible to provide these to `PEtabModel` through the `parameter_map` argument. E.g if we had known that *kB = 1.0*, then we would only define *kD* and *kP* as parameters we wish to fit:
+```petab1
+par_kD = PEtabParameter(:kD)
+par_kP = PEtabParameter(:kP)
+params = [par_kD, par_kP]
+nothing # hide
+```
+We then provide `parameter_map=[:kB => 1.0]` when we assembly our model:
+```petab1
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0, parameter_map=[:kB => 1.0])
+nothing # hide
+```
+
+### [Parameter bounds](@id petab_parameters_bounds)
+By default, when fitted, potential parameter values are assumed to be in the interval *[1e-3, 1e3]*. When declaring a `PEtabParameter` it is possible to change these values through the `lb` and `ub` arguments. E.g. we could use
+```petab1
+par_kB = PEtabParameter(:kB; lb=1e-2, ub=1e-2)
+```
+to achieve the more conservative bound *[1e-2, 1e2]* for the parameter *kB*.
+
+### [Parameter scales](@id petab_parameters_scales)
+
+Internally, parameters that are fitted are converted to a logarithmic scale (generally, this is considered advantageous). To prevent this conversion, the `scale=:lin` argument can be used. Here we change the scale of *kB* to linear:
+
+```petab1
+par_kB = PEtabParameter(:kB; scale=:lin)
+```
+
+### [Parameter priors](@id petab_parameters_priors)
+If we has prior knowledge about the distribution of a parameter, it is possible to incorporate this into the model. The prior can be any continuous, univariate, distribution from the [Distributions.jl package](https://github.com/JuliaStats/Distributions.jl). E.g we can use:
+
+```petab1
+using Distributions
+par_kB = PEtabParameter(:kB; prior=Normal(1.0,0.2))
+```
+to set a normally distributed prior (with mean `1.0` and standard deviation `0.2`) on the value of *kB*. By default, the prior is assumed to be on the linear scale of the parameter (before any potential log transform). To specify that the prior is on the logarithmic scale, the `prior_on_linear_scale=false` argument can be provided:
+```petab1
```

Comment:
Maybe a bit to nitpicking, but for a prior on parmeter (log) scale Normal is not a good choice as it has negative support, would rather use LogNormal here

Reply:
Would setting a Normal prior, and the `prior_on_linear_scale=false` be equivalent to a LogNormal prior?

Reply:
Yes, as we have if log(c) ~ N(µ, s) then c ~ LN(µ, s). 

So setting `prior=Normal(3.0, 3.0), prior_on_linear_scale=true` is equivalent to setting `prior=LogNormal(3.0, 3.0), prior_on_linear_scale=false`

Reply:
added extra note of this